style.css

```diff
@@ -250,3 +236,329 @@ body {
+}
+
+.card .card-img {
+  grid-column: 1 / 4;
+  background-image: url('./assets/images/img-placeholder.png');
+  background-size: cover;
+  background-position: center;
+  height: 200px;
+  object-fit: cover;
+}
+
+.card-content {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px;
+  background-color: #fff;
+  transition: transform 0.3s ease-out;
+  align-items: center;
+  text-align: center;
+}
+
+.card-title {
+  color: #3a4a42;
+  width: 100%;
+  height: 57px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.card .overlay h3 {
+  word-wrap: break-word;
+}
+
+/* .tags{ */
+
+.tags {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-top: 10px;
+  border-radius: 4px;
+  margin-bottom: 30px;
+  margin-left: 0;
+}
+
+.tags span.works {
+  background-color: #c6c6c6;
+  border-radius: 6px;
+  padding: 4px 8px;
+  margin-right: 5px;
+  color: black;
+}
+
+.tags span.about {
+  background-color: #fff;
+  border-radius: 6px;
+  padding: 4px 8px;
+  margin-right: 5px;
+  color: #36b37e;
+}
+
+.tags span:first-child {
+  margin-left: 0;
+}
+
+.tags span:last-child {
+  margin-right: 0;
+}
+
+.project-btn {
+  position: relative;
+  background-color: #36b37f;
+  color: #fff;
+  padding: 8px 18px;
+  border-radius: 5px;
+  border: none;
+  cursor: pointer;
+  font-size: 11px;
+  bottom: 8px;
+  width: 122px;
+  height: 48px;
+  align-items: center;
+}
+
+/* about me  */
+
+.about-me .project-btn::after {
+  content: "";
+  position: absolute;
+  top: -10px;
+  left: -150px;
+  width: 50px;
+  height: 50px;
+  background-image: url('./assets/images/about-bg-left.svg');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.header-content h1:nth-of-type(2) {
+  color: #70c572;
+}
+
+/* about me */
+.aboutme-container {
+  position: relative;
+  width: 100%;
+  max-width: 600px;
+  text-align: center;
+  z-index: 1;
+  align-items: center;
+  margin: auto;
+  padding: 0 18px;
+  margin-bottom: 50px;
+}
+
+.card-img-content {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  padding: 20px;
+  background-color: #fff;
+  transition: transform 0.3s ease-out;
+  align-items: center;
+  text-align: center;
+}
+
+/* About card */
+.about-card {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("./assets/images/img-placeholder.png");
+  background-size: 100% 100%;
+  background-position: center;
+  width: 100%;
+  height: 400px;
+  border-radius: 10px;
+  margin-top: 20px;
+}
+
+.aboutme-title {
+  background-image: url(./assets/images/about-bg-right.svg);
+  background-position: right;
+  background-repeat: no-repeat;
+  background-size: 20%;
+  z-index: 2;
+}
+
+.logo {
+  position: absolute;
+  top: 0;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+/* contact me */
+.contact-container {
+  width: 100%;
+  max-width: 600px;
+  text-align: center;
+  z-index: 1;
```

Comment:
- [ ] I would encourage you to try and make the contact container exactly as it is on the Figma template design. Observe the white spaces in the original design and make sure to replicate them in your design.
- [ ] Ensure the social icons are well-aligned by applying the `align-items: center;`.
- [ ] The `Get in Touch` button should be given the right font size, color, and properties as given in the original design.

> Figma design
<img width="256" alt="Screenshot 2023-04-29 at 13 35 21" src="https://user-images.githubusercontent.com/71140133/235302984-9e2e21e0-03b5-47a1-9357-31452dc7d98d.png">

> Your design
<img width="374" alt="Screenshot 2023-04-29 at 13 35 48" src="https://user-images.githubusercontent.com/71140133/235303007-35742bbb-e684-40ef-991a-ab3318140d80.png">